packages/studio-web/src/app/shared/wc-styling/wc-styling-helper.html

```diff
@@ -0,0 +1,201 @@
+<div class="style-helper">
+  <p>
+    To customize the look and feel of your read-along, you need to upload or
+    write some Cascading Style Sheets in the styling section.
+  </p>
+  <p>
+    We are providing a table below to help you customize. Please note that your
+    design must include both the light (<code>.theme--light</code>) and dark
+    (<code>.theme--dark</code>) themes
+  </p>
+
+  <h2>Common classes</h2>
+  <table>
+    <thead>
+      <tr>
+        <th>Class</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th>
+          <code>.sentence__word</code>
+        </th>
+        <td>The aligned text</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence__word.reading</code>
+        </th>
+        <td>The aligned text being read</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence__text</code>
+        </th>
+        <td>The unaligned text</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence__translation</code>
+        </th>
+        <td>The translated text</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence</code>
+        </th>
+        <td>The sentence container</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.paragraph</code>
+        </th>
+        <td>The paragraph containing the</td>
+      </tr>
+    </tbody>
+  </table>
+  <h2>Sample Style</h2>
+  <!-- prettier-ignore -->
+  <pre class="css-block">
+    <span class="css-class">:root,
+#read-along-container</span> &#123;
+  --theme-light-foreground: rgb(80, 70, 70) !important;
+  --theme-light-background: rgb(250, 240, 240) !important;
+  --theme-light-system-background: rgb(240, 230, 230) !important;
+  --theme-light-system-foreground: rgb(70, 60, 60) !important;
+  --theme-dark-foreground: rgb(250, 242, 242) !important;
+  --theme-dark-background: rgb(90, 80, 80) !important;
+  --theme-dark-system-background: rgb(80, 70, 70) !important;
+  --theme-dark-system-foreground: rgb(240, 232, 232) !important;
+&#125;
+
+<span class="comment">
+    /**
```

Comment:
All the open comment lines with `    /**` should be 0-indented, again so they show correctly in the help.